<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changeWordSentimentForm</class>
 <widget class="QMainWindow" name="changeWordSentimentForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Изменить тональность слова</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="rightMargin">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>77</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="1" column="1" colspan="2">
            <widget class="QLabel" name="labelError">
             <property name="minimumSize">
              <size>
               <width>251</width>
               <height>16</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>251</width>
               <height>16</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>7</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 0, 0);</string>
             </property>
             <property name="text">
              <string>asdasd</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="labelWord">
             <property name="minimumSize">
              <size>
               <width>51</width>
               <height>21</height>
              </size>
             </property>
             <property name="text">
              <string>Слово:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLineEdit" name="wordEdit">
             <property name="minimumSize">
              <size>
               <width>261</width>
               <height>31</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>Введите слово</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>37</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGroupBox" name="chooseSentimentGroupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>201</width>
             <height>161</height>
            </size>
           </property>
           <property name="title">
            <string>Тональность</string>
           </property>
           <widget class="QRadioButton" name="radioButtonPSTV">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>30</y>
              <width>131</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Положительная</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QRadioButton" name="radioButtonNGTV">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>70</y>
              <width>131</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Отрицательная</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="radioButtonNEUT">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>110</y>
              <width>121</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Нейтральная</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="buttonChange">
             <property name="minimumSize">
              <size>
               <width>91</width>
               <height>31</height>
              </size>
             </property>
             <property name="text">
              <string>Изменить</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonCancel">
             <property name="minimumSize">
              <size>
               <width>91</width>
               <height>31</height>
              </size>
             </property>
             <property name="text">
              <string>Отмена</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>467</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
